apps/examples/testcase/le_tc/filesystem/tc_fs_procfs.c

```diff
@@ -266,8 +266,15 @@ void tc_fs_procfs_main(void)
 {
 	int ret;
 	struct stat st;
-
+	
 	ret = mount(NULL, PROC_MOUNTPOINT, "procfs", 0, NULL);
+	if (ret == EEXIST) {
+		ret = umount(PROC_MOUNTPOINT);
+		TC_ASSERT_EQ("umount", ret, OK);
+
+		ret = mount(NULL, PROC_MOUNTPOINT, "procfs", 0, NULL);
+	}
```

Comment:
I think this is not right with respect to procfs automount. you should mount procfs only when `CONFIG_FS_AUTOMOUNT_PROCFS` is disabled. 
i.e.
```
#ifndef CONFIG_FS_AUTOMOUNT_PROCFS
    ret = mount(NULL, PROC_MOUNTPOINT, "procfs", 0, NULL);
    .
    .
#endif
```

Reply:
When procfs automount is enabled, User assume that there will not be "/proc" mount point. But when `CONFIG_FS_AUTOMOUNT_PROCFS`  is disabled and fs tc is executed, "/proc" remains mounted. Is it right?